tinymce/settings.py

```diff
@@ -13,9 +13,12 @@
 
 if 'staticfiles' in settings.INSTALLED_APPS or 'django.contrib.staticfiles' in settings.INSTALLED_APPS:
     JS_URL = getattr(settings, 'TINYMCE_JS_URL',os.path.join(settings.STATIC_URL, 'tiny_mce/tiny_mce.js'))
-    JS_ROOT = getattr(settings, 'TINYMCE_JS_ROOT',os.path.join(settings.STATIC_ROOT, 'tiny_mce'))
+    JS_ROOT = getattr(settings, 'TINYMCE_JS_ROOT', None)
```

Comment:
It is what powers the findstatic command: https://docs.djangoproject.com/en/1.8/ref/contrib/staticfiles/#findstatic